pennylane/templates/subroutines/select.py

```diff
@@ -1086,3 +1086,91 @@ def _select_decomp_unary(*_, ops, control, work_wires, partial, **__):
 
 
 add_decomps(Select, _select_decomp_unary)
+
+
+# Decomposition of Select using one work wire to control the target operations
+
+
+def _select_resources_multi_control_work_wire(op_reps, num_control_wires, num_work_wires, partial):
```

Comment:
Suggest putting "resources" at the end
```suggestion
def _select_multi_control_work_wire_resources(op_reps, num_control_wires, num_work_wires, partial):
```